.github/workflows/ci.yml

```diff
@@ -21,32 +21,22 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v4
 
-      - name: Setup pnpm
-        uses: pnpm/action-setup@v2.4.0
+      - name: Setup bun
+        uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: latest
 
       - name: Setup Node 18
         uses: actions/setup-node@v3
         with:
           node-version: 18
 
-      - name: Get pnpm store directory
-        id: pnpm-cache
-        run: |
-          echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
-
-      - name: Setup pnpm cache
-        uses: actions/cache@v3
-        with:
-          path: ${{ steps.pnpm-cache.outputs.pnpm_cache_dir }}
-          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
-          restore-keys: |
-            ${{ runner.os }}-pnpm-store-
```

Comment:
Stupidly fast without even caching (or maybe the action does this automatically) 
<img width="1402" alt="CleanShot 2023-09-15 at 09 14 45@2x" src="https://github.com/t3-oss/create-t3-turbo/assets/51714798/0a6fd5f9-7cbf-4908-99bc-5cdf4aa32280">